go.mod

```diff
@@ -4,7 +4,15 @@ go 1.22.1
 
 require (
 	github.com/bwmarrin/discordgo v0.27.1 // indirect
+	github.com/cucumber/gherkin/go/v26 v26.2.0 // indirect
+	github.com/cucumber/godog v0.14.0 // indirect
```

Comment:
did you run `go mod tidy` before committing? I expect this to be a direct dependency

Reply:
nope, thanks!